### PR TITLE
Added parquet benchmarks for features per day

### PR DESCRIPTION
**Pull request by vlulla**
Now we have same queries for zarr and parquet.

## Overview

The parquet and zarr benchmarks now have the same queries over the same time ranges.

### Checklist

- [x] Ran `nbautoexport export .` in `/opt/src/notebooks` and committed the generated scripts. This is to make reviewing notebooks easier. (Note the export will happen automatically after saving notebooks from the Jupyter web app.)
- [x] Documentation updated if needed
- [x] PR has a name that won't get you publicly shamed for vagueness

### CODE REVIEW

File: src/esip-2022-presentation/script/benchmark_queries.py [hunk 1]
```diff
@@ -145,19 +146,29 @@ def get_json(uri):
...
-query_map = {'mean_day': query_map['mean_day']}
+nb_repeats = 5
+## query_map = {'mean_day': query_map['mean_day']}
 time_ranges = [
     slice('1990-01-01', '1990-02-01'),
+    slice('1990-01-01', '1991-01-01'),
+    slice('1990-01-01', '2000-01-01'),
 ]
+nb_reaches = len(avail_comids)
+nb_workers = len(client.scheduler_info()['workers'])
```
Comment (lewfish) [inline src/esip-2022-presentation/script/benchmark_queries.py:157]: It seems that this block of code is duplicated from above, and doesn't actually run a small set of experiments as the comment says.

**Review by lewfish — CHANGES_REQUESTED** [review]

**Comment by vlulla** [comment]
Hmmm...I copy/pasted into a new ipynb only the necessary input blocks (1, 2, 3, 4, 7, 8, and 14) and the `results` dataframe (from block 14) states that there were only 1787 `nb_reaches` (which corresponds to `nb_reaches` in the zarr queries). What am I missing?

**Comment by lewfish** [comment]
> Hmmm...I copy/pasted into a new ipynb only the necessary input blocks (1, 2, 3, 4, 7, 8, and 14) and the `results` dataframe (from block 14) states that there were only 1787 `nb_reaches` (which corresponds to `nb_reaches` in the zarr queries). What am I missing?

I think that's the value it's supposed to have. What did you expect it to be?

**Comment by vlulla** [comment]
Oh I see. I have to remove the comment above the block! Will do...

File: src/esip-2022-presentation/script/benchmark_queries.py (hunk 1 above)
Comment (vlulla) [reply src/esip-2022-presentation/script/benchmark_queries.py:157]: Removed the incongruent comment.

**Comment by vlulla** [comment]
Had to do `git commit --amend` because my jupyter notebook has stopped working locally and it did not export the python script! So, manually fixed it manually.

**Review by lewfish — CHANGES_REQUESTED** [review]
Do you know why there are two commits with the same name? I think should should squash them together using an interactive rebase. Also, if you're not filling in part of the PR template because it doesn't apply, you should just delete the irrelevant parts.

**Comment by vlulla** [comment]
@lewfish I did a `git rebase -i` to squash multiple commits on my local branch and git now says that my local branch and origin have diverged. Is it alright to do a `git push --force` now?

Removed the extraneous verbiage from the first comment!

**Comment by lewfish** [comment]
> Is it alright to do a `git push --force` now?

yes

**Comment by vlulla** [comment]
Fixed everything you suggested! Can you please review it?

File: src/esip-2022-presentation/script/benchmark_queries.py [hunk 2]
```diff
@@ -143,21 +144,29 @@ def get_json(uri):
...
```
Comment (lewfish) [inline src/esip-2022-presentation/script/benchmark_queries.py:155]: Why does this cell of code needs to be here at all?

**Review by lewfish — CHANGES_REQUESTED** [review]

File: src/esip-2022-presentation/script/benchmark_queries.py (hunk 2 above)
Comment (vlulla) [reply src/esip-2022-presentation/script/benchmark_queries.py]: Removed it!

**Review by lewfish — APPROVED** [review]

**Comment by vlulla** [comment]
Squashed the two commits into one. Thanks for the patient review!

**Pull request merged**